locale/ar/translations.json

```diff
@@ -682,8 +682,8 @@
   "Narrow to private messages.": "تضييق العرض للرسائل الخاصة.",
   "Narrow to starred messages.": "تضييق العرض إلى الرسائل المميزة بنجمة.",
   "Narrow to stream": "تضييق العرض للغرفة",
-  "Narrow to stream &quot;{display_recipient}&quot;": "تضييق العرض للغرفة &quot;{display_recipient}&quot;",
-  "Narrow to stream &quot;{display_recipient}&quot;, topic &quot;{topic}&quot;": "تضييق العرض للموضوع &quot;{display_recipient}&quot; ، الموضوع &quot;{topic}&quot;",
+  "Narrow to stream &quot;{display_recipient}&quot;": "تضييق العرض للغرفة \"{display_recipient}\"",
+  "Narrow to stream &quot;{display_recipient}&quot;, topic &quot;{topic}&quot;": "تضييق العرض للموضوع \"{display_recipient}\" ، الموضوع \"{topic}\"",
```

Comment:
This is definitely wrong. From context, this value must be HTML-escaped.

In general, we need to look at the context to decide whether a value should be escaped; a script can’t decide for us. This is crazy unsafe and we should definitely fix things so it’s possible for a script to decide, but right now it’s not.

[Discussion](https://chat.zulip.org/#narrow/stream/58-translation/topic/removing.20HTML.20escapes.20left.20behind.20by.20Transifex.20MT.20.286.2Ex.29/near/1486122.2E01)